cirq/google/api/v2/results.py

```diff
@@ -17,6 +17,7 @@ class MeasureInfo(
             ('key', str),
             ('qubits', List[devices.GridQubit]),
             ('slot', int),
+            ('mask', List[bool]),
```

Comment:
I'd suggest calling this `invert_mask` to be consistent with usage in `MeasurementGate` itself.